### PR TITLE
Add client support for updating replicas.

### PR DESCRIPTION
**Pull request by atoaca**

### CODE REVIEW

File: coriolisclient/cli/replicas.py [hunk 1]
```diff
@@ -225,3 +225,58 @@ def get_parser(self, prog_name):
     def take_action(self, args):
         obj_list = self.app.client_manager.coriolis.replicas.list()
         return ReplicaFormatter().list_objects(obj_list)
+
+
+class UpdateReplica(show.ShowOne):
+    """Create a new replica"""
+    def get_parser(self, prog_name):
+        parser = super(UpdateReplica, self).get_parser(prog_name)
+        parser.add_argument('id', help='The replica\'s id')
```
Comment (aznashwan) [inline coriolisclient/cli/replicas.py:234]: It looks to me like the `id` is optional (which it isn't), could you please add a `required=True` here?

File: coriolisclient/v1/replicas.py [hunk 2]
```diff
@@ -79,3 +79,14 @@ def delete_disks(self, replica):
...
+    def update(self, replica, updated_values):
+        data = {
+            "replica": updated_values
+        }
+        response = self._put(
+            '/replicas/%s' % base.getid(replica), data)
+        response = response.to_dict()
+
+        return replica_executions.ReplicaExecution(
+            self, response.get("execution"), loaded=True)
```
Comment (aznashwan) [inline coriolisclient/v1/replicas.py]: Why does the above `delete_disks` call need to call `.json()` on the `response` and this doesn't?

Comment (atoaca) [reply coriolisclient/v1/replicas.py]: Fixed, was using the incorrect method before

File: README.rst [hunk 3]
```diff
@@ -158,6 +158,14 @@ The process of creating replicas is similar to starting migrations::
     --destination-environment "$DESTINATION_ENV" \
     --instance $VM_NAME
 
+Updating a replica
+------------------
+
+To update a replica::
+
+    coriolis replica update  $REPLICA_ID \
+    --destination-environment "$DESTINATION_ENV"
```
Comment (aznashwan) [inline README.rst:167]: Could you please also add the `--source_environment`, `--notes`, `--network_map` and all the storage-related params (namely `--default-storage-backend`, `--disk-storage-mapping` and `--storage-backend-mapping`) for the example?

**Review by aznashwan — CHANGES_REQUESTED** [review]
Please update the README for the full monty of options.

File: README.rst (hunk 3 above)
Comment (aznashwan) [reply README.rst:167]: Please ignore this comment as ideally we should make a separate PR to add all the new args.

**Review by aznashwan — APPROVED** [review]

**Comment by aznashwan** [comment]
LGTM, merging, many thanks!

**Pull request merged**